robottelo/ui/rhai.py

```diff
@@ -16,3 +17,19 @@ def view_registered_systems(self):
             locators['insights.registered_systems']
         ).text
         return result
+
+    def unregister_system_from_inventory(self, system_name, confirm=True):
+        """Unregister system from inventory"""
+        Navigator(self.browser).go_to_insights_inventory()
+        # select the system
+        self.assign_value(
+            locators["insight.inventory.system_checkbox"] % system_name,
+            True
+        )
+        # click on drop-down actions button
+        self.click(locators["insight.inventory.actions_button"])
+        # click on unregister action
+        self.click(locators["insight.inventory.action_unregister"])
+        if confirm:
```

Comment:
And if `confirm` is False? Is there any modal window which we'll get stuck on? I think you should add some click on `cancel` button if confirm == False.